.clang-format

```diff
@@ -20,7 +20,6 @@ BraceWrapping:
   AfterObjCDeclaration: true
   AfterStruct:     true
   AfterUnion:      true
-  AfterExternBlock: false
```

Comment:
Please no Clang-Format changes in this PR.  Let us not creep the scope and only tackle setting up initial cmake support.